custom_mutators/honggfuzz/input.h

```diff
@@ -77,11 +77,11 @@ static inline uint64_t util_rndGet(uint64_t min, uint64_t max) {
 }
 static inline uint64_t util_rnd64() { return rand_below(afl_struct, 1 << 30); }
 
-static inline size_t input_getRandomInputAsBuf(run_t *run, const uint8_t **buf) {
-  *buf = queue_input;
+static inline const uint8_t* input_getRandomInputAsBuf(run_t* run, size_t* len) {
```

Comment:
Where do the left-aligned pointers come from? Is that the normal honggfuzz style?

Reply:
yes. afaik the custom mutators are not formatted because it breaks either honggfuzz or (and?) libfuzzer compilation